.github/workflows/ci.yml

```diff
@@ -153,10 +153,49 @@ jobs:
           service_url: '${{ env.INTEGRATION_SERVICE_URL }}'
           requested_permissions: '{"scope":"integ","repositories":["github-token-minter"],"permissions":{"issues":"read"}}'
 
-      - name: 'verify'
+      - name: 'verify-github-token'
         run: |
           curl --fail \
             -H "Accept: application/vnd.github+json" \
-            -H "Authorization: Bearer ${{ steps.mint-token.outputs.token }}"\
+            -H "Authorization: Bearer ${{ steps.mint-github-token.outputs.token }}"\
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/abcxyz/github-token-minter/issues/events
+
+      - id: 'auth'
+        name: 'Authenticate to Google Cloud'
+        uses: 'google-github-actions/auth@71fee32a0bb7e97b4d33d548e7d957010649d8fa' # ratchet:google-github-actions/auth@v2
+        with:
+          create_credentials_file: false
+          export_environment_variables: false
+          workload_identity_provider: '${{ vars.wif_provider }}'
+          service_account: '${{ vars.wif_service_account }}'
+          token_format: 'id_token'
+          id_token_audience: '${{ env.INTEGRATION_SERVICE_AUDIENCE }}'
+          id_token_include_email: true
+
+      - id: 'get-oidc-caller-token'
+        name: 'Get GCloud Auth Token to use as OIDC token input to token minter'
+        uses: 'google-github-actions/auth@71fee32a0bb7e97b4d33d548e7d957010649d8fa' # ratchet:google-github-actions/auth@v2
+        with:
+          create_credentials_file: false
+          export_environment_variables: false
+          workload_identity_provider: '${{ vars.wif_provider }}'
+          service_account: '${{ vars.wif_service_account }}'
+          token_format: 'id_token'
+          id_token_audience: 'abcxyz/github-token-minter'
+          id_token_include_email: true
+
+      - id: 'mint-and-verify-gcloud-token'
+        run: |
+          token=$(curl --fail \
+            -X POST \
+            -H "Content-Type: application/json" \
+            -H "X-GitHub-OIDC-Token: ${{ steps.get-oidc-caller-token.outputs.id_token }}" \
```

Comment:
As a follow up, does it make sense to still look for a header named `X-GitHub-OIDC-Token`? Maybe just rename it to X-OIDC-Token?

@bradegler thoughts?

Reply:
I think we would need to update the action then too.

Reply:
Happy to do this as a followup if we want! Resolving this now so I can merge :)

Reply:
Let's leave the existing and add a new one. I don't think we have anything that calls minty outside of the action but just in case I'd rather be backwards compatible.